node/_tools/list_remaining_tests.ts

```diff
@@ -0,0 +1,95 @@
+// Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
+import { walk } from "../../fs/walk.ts";
+import { withoutAll } from "../../collections/without_all.ts";
+
+// deno-lint-ignore no-explicit-any
+type Object = Record<string, any>;
+
+const encoder = new TextEncoder();
+
+const NODE_API_BASE_URL = "https://api.github.com/repos/nodejs/node";
+const NODE_BASE_URL = "https://github.com/nodejs/node";
+const NODE_IGNORED_TEST_DIRS = [
+  "addons",
+  "async-hooks",
+  "cctest",
+  "doctool",
+  "embedding",
+  "fixtures",
+  "fuzzers",
+  "js-native-api",
+  "node-api",
+  "overlapped-checker",
+  "report",
+  "testpy",
+  "tick-processor",
+  "tools",
+  "v8-updates",
+  "wasi",
+  "wpt",
+];
+
+async function getNodeTestDirSHA(): Promise<string> {
+  const response = await fetch(NODE_API_BASE_URL + "/contents");
+  const body = await response.json();
+  return body
+    .find(({ name }: Object) => name === "test")
+    .sha;
+}
+
+async function getNodeTests(sha: string): Promise<string[]> {
+  const url = NODE_API_BASE_URL + "/git/trees/" + sha + "?recursive=1";
+  const response = await fetch(url);
+  const body = await response.json();
+
+  return body.tree
+    .filter(({ path }: Object) =>
+      path.includes("test") && path.endsWith(".js") &&
```

Comment:
I think you could tighten this check up a little bit:

```suggestion
      path.includes("/test-") && path.endsWith(".js") &&
```